Source/santad/EventProviders/MockFAAPolicyProcessor.h

```diff
@@ -24,13 +24,20 @@
 
 #import "Source/common/SNTCachedDecision.h"
 #include "Source/santad/DataLayer/WatchItemPolicy.h"
+#include "Source/santad/EventProviders/EndpointSecurity/Enricher.h"
+#include "Source/santad/Logs/EndpointSecurity/Logger.h"
```

Comment:
```suggestion
#import "Source/santad/EventProviders/EndpointSecurity/Enricher.h"
#import "Source/santad/Logs/EndpointSecurity/Logger.h"
```

These are both objc++

Reply:
Deferring.